design2.css

```diff
@@ -0,0 +1,136 @@
+body {
+	background-color: #008800;
+	font-family: Verdana, Arial, Helvetica, sans-serif;
+	text-align: center;
+}
+* {
+	margin: 0px;
+	padding: 0px;
+}
+html {height:100%; margin-bottom:1px;} /* Forces vertical scroll bar to be visible */
+#wrapper {
+	width: 900px;
+	margin-right: auto;
+	margin-left: auto;
+	background-image: url(images/Column1.jpg);
+	background-repeat: repeat-y;
+	text-align: left;
+	background-color: #FFFFFF;

```

Comment:
This is a minor information and no change needed, `#FFFFFF` is all uppercase letters the default Dom handles small case letters but modern browsers can easily translate this so not a problem.